.github/workflows/buildall.yml

```diff
@@ -0,0 +1,43 @@
+name: PlatformIO CI

```

Comment:
Can you clarify a bit on what we can do with this? Is there something I need to setup?

Reply:
Have a look here: https://github.com/hogthrob/ESP32-Rhasspy-Satellite/actions

With this workflow, Github will attempt to build all the devices defined in the workflow "binary" list for you. You can tell at a glance if this worked out. You can also look at the resulting log of each individual  build. It happen (at least I configured it this way, on each push to the repository or if someone creates a pull request for your repository.

You (and also the author of the Inmp441 commit) would have spotted right away, that this code does not build. It is especially useful to make sure that a commit does not break any of the existing device builds.

How to use this:
I think you need to enable "Actions" for your project in the Github project settings. 
For more info, see also the help when selecting the "Actions" tab in your repository view. Have fun.

Reply:
This works, thank you for the addition! Very useful indeed